ReactiveXComponent/WebSocket/XCApiProperties.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ReactiveXComponent.WebSocket
+{
+    public class XCApiProperties
```

Comment:
Could you rename this to something like _GetXcApiRequest_? And also add the **RequestId** field.